### PR TITLE
[stable/fluent-bit] added config option to specific path on elastic output

### PR DESCRIPTION
**Pull request by 27ascii**
#### What this PR does / why we need it:
added config option to specific path on elastic output as defined in https://github.com/fluent/fluent-bit-docs/blob/master/output/elasticsearch.md

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by asherf** [comment]
@27ascii the new param should be documented in the [readme](https://github.com/helm/charts/blob/master/stable/fluent-bit/README.md)

**Review by hectorj2f — COMMENTED** [review]
/ok-to-test

File: stable/fluent-bit/values.yaml [hunk 1]
```diff
@@ -54,6 +54,8 @@ backend:
     logstash_prefix: kubernetes_cluster
     replace_dots: "On"
     time_key: "@timestamp"
+    # Optional: Path prefix for ElasticSearch
```
Comment (hectorj2f) [inline stable/fluent-bit/values.yaml:57]: Yeah, let's document this new property

**Review by hectorj2f — CHANGES_REQUESTED** [review]

**Pull request closed without merging**